src/ModelFactory.php

```diff
@@ -240,7 +240,7 @@ final protected function addState($attributes = []): self
     }
 
     /** @psalm-return class-string<TModel> */
-    abstract protected static function getClass(): string;
+    abstract public static function getClass(): string;
```

Comment:
Not happy about that change. Maybe you have a better idea how to access the entities class names?

Reply:
This is also a BC break.

It's kind of crappy but the only thing I can think of is adding a separate public static method that just calls the protected one. Marked as final and `@internal`.